application/controllers/Hook.php

```diff
@@ -13,9 +13,7 @@ public function add()
         $this->load->model('Test');
 
         //is there a GCP URL in environment variable ?
-    if (getenv('QANB_GCPURL') !== false && strpos(getenv('QANB_GCPURL'), 'http') !== false) {
-            $this->GCPURL = getenv('QANB_GCPURL');
-        }
+        $this->GCPURL = getenv('QANB_GCPURL') !== false ? getenv('QANB_GCPURL').'reports/' : 'https://storage.googleapis.com/prestashop-core-nightly/reports/';
```

Comment:
Duplicate from Home.php

Reply:
So I changed it and now it's a config value, which can be overwritten through env variables.